css/default.scss

```diff
@@ -5,7 +5,7 @@
 
 /// Variable definitions
 
-$navbar-height: 50px;
+$navbar-height: 51px;
```

Comment:
Maybe `50px + $navbar-border-bottom`? Or maybe that variable can be added to the height in the relevant rule... or maybe we should use a different `box-sizing`? I'm not sure which is the more sensible option.

Reply:
Any even number leads to that effect. (So 49, 51 are fine - the line is a thick as defined, but 48, 50 or 52 are not and show that sligthly larger border).

I tried using a `margin-top: 1px` on the scroller, but that leaves a visible gap between the navbar and the scroller, which i found to be undesireable:

![image](https://user-images.githubusercontent.com/11561719/145465081-169e708a-531e-4026-9122-6679e4bdbc80.png)

`box-sizing` should make no difference on this issue, as the border is just visibly too large but measurements are just fine.

We could also consider this negligible. I mean, it's hardly visible anyways, so that's a fair option too.

Reply:
Fixing a blunder makes sense if it's a small change, even if it's just a 1-pixel strip. If the value must be odd, then please note as such in a comment—and make sure to use a `//` comment so that it's SCSS-only.

Reply:
Alright. Thanks for the heads up on the `//`. :)